### ISSUE TITLE
Increased urinary 18-oxocortisol  level

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11446]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increase in the level of 18-oxocortisol in the urine.
PMID: 25905293 In familial hyperaldosteronism type 1 or glucocorticoid-remediable aldosteronism, urinary hybrid steroids 18-oxocortisol and 18-hydroxycortisol are approximately 20-fold higher than in sporadic aldosteronomas. 
PMID: 29763159 Type I familial primary hyperaldosteronism is a rare, autosomal dominant, heritable disorder characterized by elevated levels of 18-oxocortisol and 18-hydroxycortisol.

**Parent term (use hpo.jax.org/app)**
Abnormal urine metabolite level HP:0033354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Type I familial primary hyperaldosteronism ORPHA:403

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
https://pmc.ncbi.nlm.nih.gov/articles/PMC4642692/
Need to restructure these metabolites first

**Comment by pnrobinson** [comment]
@carolina-f 
Added as HP:0020336
